src/sfdt/generateListLetters.ts

```diff
@@ -16,36 +16,36 @@ export const getAsLeadingZero = function(listValue: number): string {
  * @param magnitude
  * @param letter
  */
```

Comment:
@ankitaShrma lets use this: https://www.npmjs.com/package/romans to do conversion.
Or choose one from: https://www.npmjs.com/search?q=keywords:roman%20numerals

Reply:
Also can you please add example for every of what it accepts and what it returns.
Eg: 
```
generateRomanNumber(454) 
// returns:"CDLIV"
```